helpers.go

```diff
@@ -200,10 +201,10 @@ func preventImageOverwrite(repo, tag string) error {
 	imageExists, err := checkImageExists(repo, tag)
 
 	if err != nil {
-		return fmt.Errorf("Cannot proceed safely: %v.", err)
+		return fmt.Errorf("cannot proceed safely: %v", err)
 	}
 	if imageExists {
-		return fmt.Errorf("Cannot overwrite an existing image. Please use a different repository/tag.")
+		return errors.New("cannot overwrite an existing image. Please use a different repository/tag")
```

Comment:
But even without a format value it works fine, IMHO it should not be mixed. Use entirely `errors.New` or `fmt.Errorf`.